src/bin/bat/app.rs

```diff
@@ -216,8 +216,12 @@ impl App {
                 _ => VisibleLines::Ranges(
                     self.matches
                         .values_of("line-range")
-                        .map(|vs| vs.map(LineRange::from).collect())
```

Comment:
Would be great if you could de-duplicate the code for line-range and highlight-line